internal/uvm/create_wcow.go

```diff
@@ -85,6 +85,11 @@ func prepareConfigDoc(ctx context.Context, uvm *UtilityVM, opts *OptionsWCOW) (*
 	// Align the requested memory size.
 	memorySizeInMB := uvm.normalizeMemorySize(ctx, opts.MemorySizeInMB)
 
+	vmMemoryBackingType := hcsschema.MemoryBackingType_PHYSICAL
+	if opts.AllowOvercommit {
+		vmMemoryBackingType = hcsschema.MemoryBackingType_VIRTUAL
```

Comment:
Should we remove this since we're only supporting physical backing for now?

Reply:
Yes, done.